static/bidder-info/logan.yaml

```diff
@@ -0,0 +1,19 @@
+endpoint: "http://USeast2.logan.ai/pserver"
+maintainer:
+  email: "support@logan.ai"
+capabilities:
+  site:
+    mediaTypes:
+    - banner
+    - video
+    - native
+    
+  app:
+    mediaTypes:
+    - banner
+    - video
+    - native
+userSync:
+  redirect:
+    url: "https://ssp-cookie.logan.ai/pserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redir={{.RedirectURL}}"
```

Comment:
Is the URL correct? I tried the redirect in the browser and I'm getting a 404:
![image](https://user-images.githubusercontent.com/15863840/210189744-1e3e87f8-6d2d-4e5e-bebe-0841e0aa4ace.png)

Reply:
Same for me. Here is my test url: https://ssp-cookie.logan.ai/pserver?gdpr=&gdpr_consent=&us_privacy=&redir=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dlogan%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BUID%5D